api/envoy/api/v2/cds.proto

```diff
@@ -632,6 +632,30 @@ message Cluster {
   // If this flag is not set to true, Envoy will wait until the hosts fail active health
   // checking before removing it from the cluster.
   bool drain_connections_on_host_removal = 32;
+
+  // New mechanism for LB policy configuration. If set and understood
+  // by the client, replaces the lb_policy field, the common_lb_config
+  // field, and the fields in the lb_config oneof. For backward
+  // compatibility, management servers should populate both sets of fields.
```

Comment:
Should we mark the `lb_policy` field deprecated?

Reply:
Also, is someone on your team planning on doing the lifting to translate existing LB policies to the new ones in Envoy? In general, we're not super psyched about fields being added without Envoy implementation.